pyInterface/module/pyPartialWaveFit/_startParameterGenerator.py

```diff
@@ -130,11 +130,12 @@ def __init__(self, model, seed=None):
 		# build real-valued integral matrix over all sectors
 		realMatrices = []
 		self.totNmbRealAmplitudes = 0
-		for accMatrix in self.accMatrices:
-			# use accMatrix[0]
-			realAccMatrixTemp = np.zeros(shape=(2*accMatrix[0].shape[0],2*accMatrix[0].shape[0]),dtype=np.float64)
-			realAccMatrixTemp[:-1:2,:] = accMatrix[0].view(np.float64)
-			realAccMatrixTemp[1::2,:] = (1.j*accMatrix[0]).view(np.float64)
+
+		# use first an ONLY dataset: accMatrices[0]
+		for accMatrix in self.accMatrices[0]:
+			realAccMatrixTemp = np.zeros(shape=(2*accMatrix.shape[0],2*accMatrix.shape[0]),dtype=np.float64)
+			realAccMatrixTemp[:-1:2,:] = accMatrix.view(np.float64)
```

Comment:
I guess you want to get the real and imaginary parts here, right?
Why don't you use `np.real` and `np.imag`?